docs/pages/build/orbit.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Expo Orbit
+maxHeadingDepth: 4
+description: Accelerate your development workflow with one-click build launches and simulator management.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import Video from '~/components/plugins/Video';
+
+Expo Orbit for macOS makes it faster and easier to install and run builds from EAS or elsewhere, and to run Snack projects on simulators and physical devices.
```

Comment:
```suggestion
Expo Orbit for macOS makes it faster to install and run builds from EAS or run Snack projects, on simulators and physical devices.
```

- What does elsewhere here mean? Since we are introducing Orbit for the first time in our docs, let's be as explicit as we can about the use cases.
- Going to remove the phrase "easier" as we can leave that for our user base to decide. I think using the phrase "faster ..." is much more useful.
- Let's also add a pause before sims and physical device so that it indicates that both EAS builds and snacks can be used here.

Reply:
This sounds great! Thanks for the context. Yeah, having this makes it clear.